docs/md/workflow-options.md

```diff
@@ -0,0 +1,158 @@
+    workflows).
+    
+## Usage
+
+!!! info "Summary"
+    - Workflow options are an arbitrary JSON object stored in a key of `workflow_options`
+    - Can be provided either per-workflow or for an entire workload (or both)
+    - Optional when you make requests but will always be included in responses
+
+Suppose the following valid workload request that you might `POST` to `/create` or `/exec`:
+```json
+{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+  "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+  "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+  "pipeline": "ExternalWholeGenomeReprocessing",
+  "project": "PO-1234",
+  "items": [
+    {
+      "inputs": {
+        "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+        "sample_name": "TestSample1234"
+      }
+    },
+    {
+      "inputs": {
+        "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+        "sample_name": "TestSample5678"
+      }
+    }
+  ]
+}
+```
+You may optionally add arbitrary JSON objects as `workflow_options` either for individual
+workflows, for the entire workload, or both:
+```json
+{
+  "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+  "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+  "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+  "pipeline": "ExternalWholeGenomeReprocessing",
+  "project": "PO-1234",
+  "items": [
+    {
+      "inputs": {
+        "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+        "sample_name": "TestSample1234"
+      },
+      "workflow_options": {
+        "my_option": "something for sample 1234"      
+      }
+    },
+    {
+      "inputs": {
+        "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+        "sample_name": "TestSample5678"
+      },
+      "workflow_options": {
+        "my_option": "something different for sample 5678",
+        "another_option": {"foo": "bar"}      
+      }
+    }
+  ],
+  "workflow_options": {
+    "yet_another_option": "something for both of the samples"  
+  }
+}
+```
+???+ info
+    Note that for modules that don't use individual workflows
+    configured via `/create` or `/exec`, options can only be
+    configured at the workload level. For example, the All of
+    Us module configures workflows via a different endpoint
+    and thus only the per-workload field here is respected.
+
+To recap, in the above example the following workflow options will be set:
+- "my_option" will have different strings for the different samples
+- "another_option" will be an object just set for the latter sample
+- "yet_another_option" will be the same string for all samples
+
+In other words, WFL will recursively merge the options objects together to
+resolve the options for individual workflows. You can see this in WFL's
+response, which includes all workflow options:
+```json
+{
+  // Some fields omitted for brevity!
```

Comment:
What I would give for universal support of `.jsonc`. I feel like this is a helpful note but if everyone's parsers are going to throw a fit we can remove it

Reply:
I like the comments, but it does not persuade me to switch to jsonc which means additional conversions before I can use any JSON. What we could do here in docs is moving this comment out of the JSON block and wrap it in a special block?